.nix/services/hasura.nix

```diff
@@ -1,6 +1,30 @@
-{ metadata, database, graphql-port }: {
+{ pkgs, metadata, database, graphql-port }:
+let
+  context = let
+    files = pkgs.linkFarm "context" [
+      {
+        name = "metadata";
+        path = metadata;
+      }
+      {
+        name = "Dockerfile";
+        path = pkgs.writeText "Dockerfile" ''
+          FROM hasura/graphql-engine:v2.12.0-beta.1.cli-migrations-v3
+          COPY metadata /hasura-metadata
```

Comment:
https://github.com/NixOS/nixpkgs/blob/578f231a53bfbedfad4b4cc17547f67ed99a72ba/pkgs/build-support/docker/examples.nix#L27 ?

Reply:
I guess metadata can be made other derivation which is mounted here?

Reply:
it's arion gere

Reply:
Here*

Reply:
oh, my bad, sorry :)

Reply:
i though they integrated, likely in future:)

Reply:
Yes hopefully :). In any case even if we copy store paths you need a way to have the path being copied in a target if the tool has no env variable to set an arbitrary one